features/nanostack/mbed-mesh-api/source/WisunInterface.cpp

```diff
@@ -542,6 +547,33 @@ mesh_error_t WisunInterface::read_mac_statistics(mesh_mac_statistics_t *statisti
     return ret_val;
 }
 
+mesh_error_t WisunInterface::get_info(router_information_t *info_ptr)
+{
+    rpl_dodag_info_t dodag_ptr = {0};
+    uint8_t global_address[16] = {0};
+
+    if (info_ptr == NULL) {
+        return MESH_ERROR_PARAM;
+    }
+
+    int status = rpl_read_dodag_info(&dodag_ptr, current_instance_id);
```

Comment:
You need to read the instance id using RPL functions as it might change

Reply:
Should I read instance ID like this ?
https://github.com/ARMmbed/mbed-os/blob/72fa02d92bc07adfb8da6a229b39ed33e8f951ce/features/nanostack/sal-stack-nanostack/source/6LoWPAN/Bootstraps/Generic/protocol_6lowpan.c#L696-L725

Reply:
@kjbracey-arm Could you help us a bit what is the rule for selecting the instance id is this the correct way or is there something simpler?